modules/nw-multus-bridge-object.adoc

```diff
@@ -82,25 +87,24 @@ plugin:
 |Optional: Enables mac spoof check, limiting the traffic originating from the container to the mac address of the interface. The default value is `false`.
 |====
 
+ifndef::microshift[]
 [NOTE]
 ====
 The VLAN parameter configures the VLAN tag on the host end of the `veth` and also enables the `vlan_filtering` feature on the bridge interface.
 ====
 
 [NOTE]
 ====
-To configure uplink for a L2 network you need to allow the vlan on the uplink interface by using the following command:
+To configure uplink for an L2 network, you must allow the VLAN on the uplink interface by using the following command:
```

Comment:
@pmtk Is this a true statement? I haven't configure bridge VLAN for my testing yet. VLAN is one type of bridge configuration, but is not MUST action.

Reply:
@weliang1 That text does not appear in the MicroShift doc, see https://74823--ocpdocs-pr.netlify.app/microshift/latest/microshift_networking/microshift_multiple_networks/microshift-cni-multus.html#microshift-nw-multus-bridge-config-example_microshift-cni-multus. It is right above the Bridge CNI plugin configuration example in the OCP docs. I simply updated that portion for docs style, but it will only be in the OCP docs.

Reply:
Check the vlan from _Table 2. Bridge CNI plugin JSON configuration object_ in above [OCP docs](https://74823--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-additional-network.html#nw-multus-bridge-config-example_configuring-additional-network)
it is said: `Optional: Specify a virtual LAN (VLAN) tag as an integer value. By default, no VLAN tag is assigned`.

Reply:
@weliang1 I did not change the meaning of anything, so this is really out of scope of the PR. I'll see if I can find an answer, though. Everything MicroShift looks OK here?

Reply:
@ShaunaDiaz  I got the point now, the MUST word is for configuring uplink for an L2 network only.
/lgtm now

Reply:
@s1061123 or @pliurh Do either of you know the answer to this potential discrepancy in the OCP docs regarding bridge + VLAN configuration?